salt/_pillar/metalk8s.py

```diff
@@ -15,6 +15,7 @@
 
 DEFAULT_POD_NETWORK = '10.233.0.0/16'
 DEFAULT_SERVICE_NETWORK = '10.96.0.0/12'
+DEFAULT_MTU = 1460
```

Comment:
That's the default Calico uses based on GCP. Should we keep it (to smooth upgrades), or find one that better suits our use cases? (for on-prem, I don't what the lowest reasonable MTU we'd find, but maybe we could start by considering the ones on Scality Cloud - 1500 apparently)

Reply:
Don't really know, but to me it's not really part of this PR, but indeed it's a good question

Reply:
Let's keep it for now, shouldn't hurt unless we start deploying in envs where this default value is too low.